resources/guidelines/testing/store/quality-guidelines-apps/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 nav:
-  title: Quality guidelines for apps in the plugin system
+  title: Quality guidelines for apps and themes in the app system
   position: 10
 
 ---
 
-# Quality Guidelines for the Plugin System in the Shopware Store
+# Quality Guidelines for apps and themes app system in the Shopware Store
```

Comment:
No. We differentiate between apps  based on the plugin system and apps based on app system. so an app could be both: a cloud-app or a plugin-app.

therefor the navigation had to chang as we have 2 different documents with the same nav title.

Reply:
I recognized a minor grammar mistake.
The headline should be named like this:
Quality Guidelines for apps and themes based on the app system in the Shopware Store

It would be nice if you could fix that before approving. Thanks.